intake_esm/aggregate.py

```diff
@@ -179,6 +181,13 @@ def concat_ensembles(dsets, member_ids=None, join='inner'):
         attrs['history'] = new_history
     ds.attrs = attrs
 
+    # rechunk
+    if chunks is None:
```

Comment:
We have discussed potential performance implications of re-chunking. Since we are concatenating along a new dimension, it seems like in theory we could avoid re-chunking along time and impose chunks during the concat (though I am not sure that supported by xarray).

Reply:
Not 100% sure, I have the right approach in the new changes. Please check the new changes out and let me know what you think.

Reply:
this seems reasonable.